userspace/libsinsp/test/libsinsp_test_var.h.in

```diff
@@ -2,3 +2,5 @@
 
 // Absolute path to the scap files dir
 #define LIBSINSP_TEST_SCAP_FILES_DIR "${CMAKE_BINARY_DIR}/scap_files/"
+
+#define LIBSINSP_TMP_TEST_SCAP_FILES_DIR "${CMAKE_CURRENT_SOURCE_DIR}/resources/"
```

Comment:
Before merging it would be great to push this scap-file with the others. @FedeDP probably you are the one with the right permissions

Reply:
Done!